components/terrain/world.cpp

```diff
@@ -70,6 +70,12 @@ float World::getHeightAt(const osg::Vec3f &worldPos)
     return mStorage->getHeightAt(worldPos);
 }
 
+void World::clearCache()
+{
+    mTextureManager->clearCache();
+    mChunkManager->clearCache();
```

Comment:
Hm, actually this function is redundant. TextureManager and ChunkManager are added to the ResourceSystem, so are already cleared through there.